includes/Context.php

```diff
@@ -323,13 +321,14 @@ public function is_amp() {
 	 *                     false if AMP not active, or unknown mode
 	 */
 	public function get_amp_mode() {
-		// If the Web Stories plugin is enabled, consider the site to be running
```

Comment:
With regard to the changes in this file, IMHO it would be a little cleaner to move the non-web stories code into a separate function, say `get_amp_mode_from_amp_plugin()`, and then this function could be simplified along these lines. What do you think?

```php
	public function get_amp_mode() {
		$amp_mode = $this->get_amp_mode_from_amp_plugin();

		if ( false === $amp_mode ) {
			// If the Web Stories plugin is enabled, consider the site to be running
			// in Secondary AMP mode.
			if ( defined( 'WEBSTORIES_VERSION' ) ) {
				return self::AMP_MODE_SECONDARY;
			}
		}

		return $amp_mode;
	}